.spelling

```diff
@@ -391,6 +391,7 @@ playbook
 playbooks
 plugin
 plugins
+Podman
```

Comment:
Do we need both cases (podman and Podman)?

Reply:
I suppose it's a proper noun, so it'll always be capitalized unless the CMD is invoked and commands are ignored by the spell check.